simphony/cuds/simulation.py

```diff
@@ -0,0 +1,42 @@
+"""Contains simulation controllers."""
+from . import CUDS
+from ..engine import get_wrapper
+
+
+class WrapperFactory(object):
+    """Creates wrappers."""
+    @staticmethod
+    def create(wrapper_name, cuds):
+        """Create a wrapper instance of the given name.
+
+        Parameters
+        ----------
+        cuds: CUDS
+            a cuds object containing simulation data
+        wrapper_name: str
+            A SimPhoNy wrapper name, should be of of registered engines.
+        """
+        wrapper = get_wrapper(wrapper_name)
+        if not wrapper:
+            raise Exception('Unknown wrapper: %s' % wrapper_name)
+
+        return wrapper(cuds=cuds)
+
+
+class Simulation(object):
+    """Represents a simulation using CUDS.
+
+    Parameters
+    ----------
+    cuds: CUDS
+        A cuds object which contains model information.
+    wrapper_name: str
+        Name of the wrapper to launch the simulation with.
+    """
+    def __init__(self, cuds, wrapper_name):
+        self._cuds = cuds
```

Comment:
I think the Simulation wrapper is supposed to hold multiple cuds objects.  (referring to https://publicwiki-01.fraunhofer.de/SimPhoNy-Project/index.php/File:Status_of_the_SimPhoNy_framework_Specifications_D1.3_and_D1.6.pdf)
According to the slides, I imagine that "self._cuds" should be a sequence of cuds objects, and after each run, the Simulation wrapper may append to "_cuds" with the output of the run.